### ISSUE TITLE
Castling is not correctly reported with Lc0 release 0.23+

### ISSUE THREAD

**Issue by skiminki** [fohristiwhirl/nibbler#67]
### Problem

Castling moves are not properly shown by the Nibbler GUI for Lc0 0.23+. I would guess that this is because the Lc0 engine has changed the info string output for FRC.

### Reproduction

- Open Nibbler 1.1.1
- Play the following moves: 1. e4 e5 2. Nf3 Nf6 3. Bc4 Bc5 *
- Press CTRL-g for "go"

### Observations

Nibbler GUI
- lc0 0.22 -- castling move (O-O) is shown as the second best move (good)
- lc0 0.23.1 -- castling move (O-O) is shown in the bottom of the move list without stats (bad)

From Nibbler log / lc0 0.22
```
< info string e1g1  (103 ) N:       5 (+ 0) (P:  3.29%) (Q:  0.08298) (D:  0.000) (U: 0.64388) (Q+U:  0.72685) (V:  0.1383) 
```
From Nibbler log / lc0 0.23.1
```
< info string e1h1  (103 ) N:       5 (+ 0) (P:  3.29%) (Q:  0.08298) (D:  0.000) (U: 0.62544) (Q+U:  0.70842) (V:  0.1383) 
... Nibbler: invalid move received!: e1h1
```

### Test environment
- Gentoo Linux
- Prebuilt Nibbler 1.1.1
- Locally built Lc0 0.22 and 0.23.1
- Net weights: 42810

### Workaround
Use Lc0 0.22

**Comment by fohristiwhirl** [comment]
Thanks. Should be possible to fix, though I've not looked at the code for ages now... wish me luck.

**Comment by fohristiwhirl** [comment]
Should be fixed in 1.1.2.

**Comment by skiminki** [comment]
Thanks for the exceptionally quick fix! Confirmed that this now works with 0.23.1 and the current 0.24-dev.
